manager/src/main/resources/application.properties

```diff
@@ -6,10 +6,26 @@ quarkus.smallrye-openapi.info-contact-url=https://gingersnap-project.io
 quarkus.smallrye-openapi.info-license-name=Apache 2.0
 quarkus.smallrye-openapi.info-license-url=https://www.apache.org/licenses/LICENSE-2.0.html
 
-quarkus.datasource.db-kind=MYSQL
-quarkus.datasource.username=gingersnap_user
-quarkus.datasource.password=password
-quarkus.datasource.reactive.url=mysql://localhost:3306/debezium
+quarkus.datasource.mysql.db-kind=MYSQL
+quarkus.datasource.mysql.health-exclude=true
+quarkus.datasource.mysql.username=gingersnap_user
+quarkus.datasource.mysql.password=password
+quarkus.datasource.mysql.reactive.url=mysql://localhost:3306/debezium
```

Comment:
I think we need to mark this one with `%dev.`. When running in a container with other DBs this one causes an exception because we have multiple URLs defined. If running during development we can just change the properties file.

Reply:
Yeah, I noticed as well. However, I am not going to worry as I think we are abandoning this approach. We at least have the PR to look at a later point if we want to come back  to this though.